src/common/com/intellij/plugins/haxe/ide/projectStructure/ui/HaxeConfigurationEditor.form

```diff
@@ -273,7 +273,7 @@
                           <text resource-bundle="com/intellij/plugins/haxe/HaxeBundle" key="haxe.openfl.arguments"/>
                         </properties>
                       </component>
-                      <component id="39470" class="com.intellij.ui.RawCommandLineEditor">
+                      <component id="39470" class="com.intellij.ui.RawCommandLineEditor" binding="myOpenFLArgumentTextField">
```

Comment:
Correct.

Thanks for the link.  I'll go answer it after we release -- even though it's an old thread.